### PR TITLE
docker setup

### PR DESCRIPTION
**Pull request by elray1**
This PR adds setup for using Docker in the project.  See the readme in the root of the repository for usage instructions/examples.

Ultimately, the results of model runs are not reliably reproducible across different systems, even using Docker. The failure of reproducibility happens inside the call to fit the LightGBM model [here](https://github.com/reichlab/flusion/blob/f4bd6e5e970606546ea5d0f9b322d0a292f94365/code/gbq/run.py#L165-L169).  This failure manifests even though the `random_state` is set, and is not resolved by setting `n_jobs=1` as recommended in some forum posts. I could not identify any other candidate settings to try changing to ensure reproducibility.

This means that attempts to check the PR by running tests will likely fail.

### CODE REVIEW

**Review by bsweger — CHANGES_REQUESTED** [review]
If you decide to merge this, I'd suggest updated the README for the Docker build command that works on macbooks with the M-series chip:

`docker build --platform linux/amd64 -t flusion .`

File: README.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ This repository has the following directories:
 
 ## Environment setup
```
Comment (matthewcornell) [inline README.md:15]: maybe goes without saying?:
```bash
cd <this repo's root>
```

**Comment by matthewcornell** [comment]
@elray1 this looks good to me. I'd say merge it!

**Comment by elray1** [comment]
Thanks for the reviews.  I updated the readme to add the command that needs to be used on macs with M-series chips.  Will merge!

**Pull request merged**